starwars/src/App.css

```diff
@@ -27,3 +27,13 @@
   font-weight: 600;
```

Comment:
Good job on the sprint challenge Inaki. 
- You successfully completed the MVP by fetching data from 3rd party API and displayed it on the webpage. You have properly split functionality into smaller components and passed necessary data as props.
- Your project is styled properly and design looks good.
- Your code is clean and Your git workflow is fluid.
Overall:
  * Able to fetch and display data (MVP)
  * Good answers
  * Clean and modular code with multiple components
  * Fluid git workflow

Possible Improvements:
* Add some kind of loading icon or information that data is beign fetched. After loading Your app it takes some time to load and render everything so it would be great fit here
* You are close to add pagination for the API data that would be probably some good stretch exercise to complete during the weekend

Inaki I can see You are pushing Yourself really hard each week. Whenever there is some kind of obstacle You are able to work through it and overcome all the challenges. I am pretty confident that You are getting better and better with React. You did great job on the sprint challenge